### DISCUSSION TITLE
Rencontre avec Maxime

### DISCUSSION THREAD

**Discussion by CollRen** [582-41E-MA/projet-web-2-CollRen#51] [Rencontres d'équipe]
@claraquintela @NabilDjerroud 
Bonjour vous deux, Maxime aimerait nous rencontrer pour voir l'avancement du projet. Vous pouver vous connecter à nouveau sur notre rencontre de tout à l'heure? Disons pour 13h?

**Comment by claraquintela** [comment]
Oui! Je suis deja online

**Reply by NabilDjerroud** [reply]
Hello les gars, je suis disponible moi aussi.
Avez vous fait une rencontre ce matin ? j'étais dispo mais personne n'a partager le lien ?

**Comment by claraquintela** [comment]
oui.
René nous a envoyé un courriel avec le lien

Em qua., 3 de jul. de 2024 às 12:11, Nabil ***@***.***>
escreveu:

**Reply by NabilDjerroud** [reply]
pff,
il l'a envoyé où ? je n'ai rien reçu de mon coté !

**Reply by CollRen** [reply]
Texte-nous la prochaine fois, je l'ai envoyé dans une discussion de github. Je vous envoies le lien pour la rencontre de tout à l'heure dans cette discussion-ci

**Comment by CollRen** [comment]
Lien pour la rencontre de 13h avec Maxime: https://us05web.zoom.us/j/86447633144?pwd=niE2s3pqMeLUuEg4R7kqDMvUdpQ1N3.1
